resources/report_issue_user_settings.json

```
@@ -0,0 +1,126 @@
+{
+    "initialize": false,
+    "pythonPath": false,
+    "onDidChange": false,
+    "defaultInterpreterPath": false,
+    "defaultLS": {
+        "defaultLSType": true
+    },
```

```suggestion
    "defaultLS": true,
```

If all subsettings are marked as true, this should work as well.